sentry_auth_google/views.py

```diff
@@ -16,27 +16,37 @@ def __init__(self, domain=None, *args, **kwargs):
     def dispatch(self, request, helper):
         access_token = helper.fetch_state('data')['access_token']
 
-        req = safe_urlopen('{0}?{1}'.format(
+        req = safe_urlopen('{0}?{1}&alt=json'.format(
             USER_DETAILS_ENDPOINT,
             urlencode({
                 'access_token': access_token,
-            }),
+            })
         ))
         body = safe_urlread(req)
         data = json.loads(body)
 
-        if not data.get('domain'):
-            return helper.error(ERR_INVALID_DOMAIN)
+        if not data.get('data'):
+            #TODO(cnuss): Propagate this error to Sentry?
+            print('Invalid response: %s' % body)
```

Comment:
I might create a special logger:

logger = logging.getLogger('sentry.auth.google')

If we're pretty confident the invalid data will basically never happen I think ``error`` is fine, otherwise I'd say ``info``.